docs/advanced/experimental/kubernetes.md

```diff
@@ -30,6 +30,11 @@ The context manager makes sure to deploy the `Flow` when entering the context an
 All Executors in the Flow should be used with `jinahub+docker://...` or `docker://...`.
 ```
 
+```{admonition} Caution
+:class: caution
+We heavily recommend you to deploy `Flow` in a separate K8s namespace by passing `k8s_namespace` into the `Flow` arguments.
```

Comment:
Actually that is not what I meant, sorry. I mean the user should not deploy it to an already existing namespace, which is used for something else like lets say 'very_valueable_production_services' ;)